src/components/SearchPage/SearchPage.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useState } from "react";
+
+import SearchHeader from "../SearchHeader/SearchHeader";
+import PokemonData from "../PokemonData/PokemonData";
+import Footer from "../Footer/Footer";
+import { fetchPokemon } from "../../utils/FetchPokemon";
+
+function InfoPage() {
+  const [pokeInfo, setPokeInfo] = useState();
+  const [loading, setLoading] = useState(false);
+
+  const getPokemon = async (query) => {
+    setLoading(true);
+    const response = await fetchPokemon(query);
+    const results = await response.json();
+    console.log(results);
+    setPokeInfo(results);
+    setLoading(false);
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**